survival_sponsorship_compassion/models/recurring_contract_group.py

```diff
@@ -0,0 +1,30 @@
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Cyril Sester, Emanuel Cino
```

Comment:
Either do it correctly, or remove the file headers (everywhere in your new code).